src/skin/tooltips.cpp

```diff
@@ -70,6 +71,10 @@ void Tooltips::addStandardTooltips() {
             << dropTracksHere
             << tr("If Vinyl control is enabled, displays time-coded vinyl signal quality (see Preferences -> Vinyl Control).");
 
+    add("big_spinny_coverart")
+            << tr("Big Spinny/Cover Art")
+            << tr("Show a big version of the Spinny or track cover art if enabled.");
```

Comment:
These new tooltips do now show up for me. I guess this is because you didn't add them to LateNight yet?

Reply:
exactly. this about the strings only, the implementation will happen elsewhere.